htdocs/user/card.php

```diff
@@ -1905,7 +1905,7 @@
 					}
 				}
 
-				if ($caneditfield && (empty($conf->multicompany->enabled) || !$user->entity || ($object->entity == $conf->entity) || ($conf->global->MULTICOMPANY_TRANSVERSE_MODE && $conf->entity == 1)))
+				if ($caneditfield && (empty($conf->multicompany->enabled) || !$user->entity || ($object->entity == $conf->entity) || ($conf->global->MULTICOMPANY_TRANSVERSE_MODE && $object->entity == 1)))
```

Comment:
This replace a feature with another.
Old version was: if MULTICOMPANY_TRANSVERSE_MODE is on, we can modify any user whatever is its entity if we are connected on entity 1.
New version means: if MULTICOMPANY_TRANSVERSE_MODE is on, we can modify the user that was created into entity 1, even if we are connected onto another entity.
The definition of MULTICOMPANY_TRANSVERSE_MODE is still not clear for the moment (it was never documented on the page https://wiki.dolibarr.org/index.php?title=Module_MultiCompany, so i don't know which behaviour we want to have with this option so i don't know if i can validate or not). @hregis can you update the wiki page to define the rule on permission that brings this option ?